public/jsx/app.jsx

```diff
@@ -1,42 +1,55 @@
 'use strict';
 
 var React = require('react');
-var Router = require('react-router');
-var Route = Router.Route;
-var RouteHandler = Router.RouteHandler;
 var Provider = require('redux/react').Provider;
 var Connector = require('redux/react').Connector;
+var bindActionCreators = require('redux').bindActionCreators;
 
 var reduxConfig = require('redux-config');
-var CardDetails = require('views/card-details');
-var CardListing = require('views/card-listing');
-var CompletePayment = require('views/complete-payment');
 var ErrorMessage = require('components/error');
 var Login = require('views/login');
+var Purchase = require('views/purchase');
+var userActions = require('user-actions');
 
 var products = require('products');
 
 
+function parseQuery(url) {
+  // TODO: replace with querystring library or something.
```

Comment:
actually, yeah, it's a small amount of code so I guess we can keep it. I'll add a direct test for it.